src/renderer/helpers/playlists.js

```diff
@@ -0,0 +1,52 @@
+export const SORT_BY_VALUES = {
+  DateAddedNewest: 'date_added_descending',
+  DateAddedOldest: 'date_added_ascending',
+  AuthorAscending: 'author_ascending',
+  AuthorDescending: 'author_descending',
+  VideoTitleAscending: 'video_title_ascending',
+  VideoTitleDescending: 'video_title_descending',
+  Custom: 'custom'
+}
+
+export function getSortedPlaylistItems(playlistItems, sortOrder, locale, reversed = false) {
+  if (sortOrder === SORT_BY_VALUES.Custom) {
+    return reversed ? playlistItems.toReversed() : playlistItems
+  }
+
+  let collator
+
+  if (
+    sortOrder === SORT_BY_VALUES.VideoTitleAscending ||
+    sortOrder === SORT_BY_VALUES.VideoTitleDescending ||
+    sortOrder === SORT_BY_VALUES.AuthorAscending ||
+    sortOrder === SORT_BY_VALUES.AuthorDescending
+  ) {
+    collator = new Intl.Collator([locale, 'en'])
```

Comment:
(Optional) I know nothing about `Intl.Collator`
But I would like to know if using a constant object be possible (map of `locale` to `Collator` objects lazily created)
(No idea how much it helps, if too complicated might not worth it

Reply:
I'll mark it as a chore, although I'm skeptical if the access time would be considerably less than the creation time of this